classes/Link.php

```diff
@@ -724,12 +724,20 @@ public function getAdminLink($controller, $withToken = true, $sfRouteParams = ar
                     $params = array_merge($params, $sfRouteParams);
                 }
                 break;
+
             case 'AdminModulesSf':
                 if (array_key_exists('route', $sfRouteParams)) {
                     return $sfRouter->generate($sfRouteParams['route'], $sfRouteParams, UrlGeneratorInterface::ABSOLUTE_URL);
                 }
                 // New architecture modification: temporary behavior to switch between old and new controllers.
-                return $sfRouter->generate('admin_module_catalog', $sfRouteParams, UrlGeneratorInterface::ABSOLUTE_URL);
+                return str_replace('index.php/', '', $sfRouter->generate('admin_module_catalog', $sfRouteParams, UrlGeneratorInterface::ABSOLUTE_URL));
```

Comment:
Can you explain this code ? You may break the back-office for merchants without Apache URL rewriting